### DISCUSSION TITLE
Which location in "Go and Get It"

### DISCUSSION THREAD

**Discussion by maxgrothe** [RoboCupAtHome/RuleBook#744] [Q&A]
3.6.3 During the Competition:
"Task 2a - Enters the dining room and goes to a location inside the room while avoiding obstacles.
Task 2b - The robot goes directly to the shelf and takes the correct object given at the beginning of the test."

In Go and Get It, should the robot move directly from its start position to the shelf or to a different location?

**Comment by ARTenshi** [comment]
This is fine: "from its start position to the shelf".
